src/components/Nav/Login/Login.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import { IoMdClose } from 'react-icons/io';
+
+import UserFormInputs from '../UserInput/UserInput';
+import UserFormBtns from '../UserButton/UserFormBtn';
+import UserRadioBtns from '../UserRadioBtns/UserFormRadio';
+
+import * as S from './LoginEle';
+
+function Login({ form, modalClose }) {
+  const [inputValues, setInputValues] = useState({ id: '', password: '' });
+
+  const handleLoginSubmit = e => {
+    e.preventDefault();
+
+    setInputValues({ id: '', password: '' });
+  };
+  const getInputValue = e => {
+    const { name, value } = e.target;
+
+    setInputValues({ [name]: value });
+  };
```

Comment:
```suggestion
  const handleLoginSubmit = e => {
    e.preventDefault();
    setInputValues({ id: '', password: '' });
  };
  
  const getInputValue = e => {
    const { name, value } = e.target;
    setInputValues({ [name]: value });
  };
```
적절한 줄바꿈으로 가독성을 더 좋게 해주시면 좋습니다 :)

Reply:
> 적절한 줄바꿈으로 가독성을 더 좋게 해주시면 좋습니다 :)

항상 까먹네요...ㅎㅎ 😂